requests/maniphest_search.go

```diff
@@ -0,0 +1,116 @@
+package requests
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/uber/gonduit/entities"
+)
+
+// ManiphestSearchRequest represents a request to maniphest.search API method.
+type ManiphestSearchRequest struct {
+	// QueryKey is builtin or saved query to use. It is optional and sets initial constraints.
+	QueryKey    string             `json:"queryKey,omitempty"`
+	// Constraints contains additional filters for results. Applied on top of query if provided.
+	Constraints *SearchConstraints `json:"constraints,omitempty"`
+	// Attachments specified what additional data should be returned with each result.
+	Attachments *SearchAttachments `json:"attachments,omitempty"`
+
+	*entities.Cursor
```

Comment:
This doesn't seem right. The cursor is part of the response, not the request.

Reply:
+1

Reply:
you actually need cursor to request a next page

Reply:
Ah, that makes sense. In that case, we shouldn't embed this and make it a regular field with an explicitly provided name like the other cases.

From what I can tell, embedding is used only for the internal request struct.

Reply:
I think server expects "after" parameter in request body and not in "cursor" object. That's why I embedded it anonymously.